tasks/StoringGroceries.tex

```diff
@@ -48,16 +45,12 @@ \subsection{Setup}
 % %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Additional rules and remarks}
 \begin{enumerate}
-	\item \textbf{Clear area:} The robot may assume that the working area is clear, (i.e. can move slightly backwards for its task).
+		\item \textbf{Deus ex Machina:} Score reduction for requesting human assistance is applied per object as follows:
+	\begin{itemize}[nosep]
+		\item Telling or pointing out to the robot where to place an object results in a 30\% point reduction for that object.
+	\end{itemize}
 
-	\item \textbf{Object distribution:} The 10 objects to be moved are evenly distributed in random fashion including
-	4 known objects,
-	4 alike objects, and
-	2 unknown objects.
-	Among these, the robot will always find
-	a heavy object,
-	a tiny object, and 
-	an amorphous object.
+	\item \textbf{Clear area:} The robot may assume that the working area is clear, (i.e. can move slightly backwards for its task).
```

Comment:
I strongly disagree with this change. It is necessary to keep fairness (even distribution among teams) and for logistical reasons. Further, the heavy and tiny objects are necessary due to bonus points (see [scoresheet](https://github.com/RoboCupAtHome/RuleBook/blob/2019/restructure/scoresheets/StoringGroceries.tex)).

Reply:
I agree that distribution has to be equal for all teams but I think that is something for the referees to decide on location and is not something that should or needs to be in the rulebook. Why not allow the referees the flexibility to decide: "Hey at this competition no team is really strong in manipulation let's use easier objects for everyone." And also if teams get stronger we do not need to update single numbers from year to year and keep more consistency in the rulebook over time

Reply:
Because scoring needs to be consistent among tests. This tests makes pick and place an object automatically worth ~100pts, so it can be taken as measure for assigning scores to other tests. Further, we are implying that each manipulation should not take more than one minute.

Besides, we need to be consistent in the three leagues (or at least between OPL and DSPL). If you leave the decision to the referees, the difficulty will vary between leagues and you have no consistent progression over the years.

Teams know:
- There are [5, 10] objects to deal with
- We must move 5
- 3 are easy to recognize (and grasp?)
- The rest are a PITA. (← all decision making enters here)

So yes, a state machine can be used, but a more robust decision making will produce better results to maximize scoring. IMHO, that setup is way too complex for a state machine.

Reply:
I don't see how this affects consistency among other tests.
Consistency among leagues I understand but don't the referees switch leagues anyway during a competition and talks are with referees of all leagues.
Consistency over the years I agree is lost a little here but definitely not entirely.
I prefer teams know:
-There are several objects on a table
-We get points for moving up to five of them
-Our robot should know which of the objects are easy to grasp for it